plugins/reolink/src/probe.ts

```diff
@@ -46,6 +46,43 @@ async function getDeviceInfo(host: string, username: string, password: string):
     return response.body?.[0]?.value?.DevInfo;
 }
 
+export async function getToken(host: string, username: string, password: string) {
```

Comment:
This block is extracted from the `getLoginParameters` method to allow separated reuse